appnav/src/main/kotlin/io/element/android/appnav/NotLoggedInFlowNode.kt

```diff
@@ -39,12 +36,11 @@ import kotlinx.parcelize.Parcelize
 class NotLoggedInFlowNode @AssistedInject constructor(
     @Assisted buildContext: BuildContext,
     @Assisted plugins: List<Plugin>,
-    private val onBoardingEntryPoint: OnBoardingEntryPoint,
     private val loginEntryPoint: LoginEntryPoint,
     private val notLoggedInImageLoaderFactory: NotLoggedInImageLoaderFactory,
 ) : BaseFlowNode<NotLoggedInFlowNode.NavTarget>(
     backstack = BackStack(
-        initialElement = NavTarget.OnBoarding,
+        initialElement = NavTarget.Root,
```

Comment:
This node could be simplified more, since there is no need for a backstack anymore, but it can stay like this.